app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -542,6 +524,20 @@ public Cursor runQuery(CharSequence charSequence) {
         }
     }
 
+    private void resetPeopleSearch() {
```

Comment:
Although the function names are self-explanatory, it'd be nice to have a javadoc comment (documentation like this really helps beginners to our project)